datadog_checks_base/datadog_checks/base/utils/http.py

```diff
@@ -170,7 +171,18 @@ def __init__(self, instance, init_config, remapper=None, logger=None):
         auth = None
         if config['password']:
             if config['username']:
-                auth = (config['username'], config['password'])
+                if config['auth_type']:
+                    auth_type = config.get('auth_type', 'basic').lower()
+                    if auth_type == 'basic':
+                        auth = requests.auth.HTTPBasicAuth(config['username'], config['password'])
+                    elif auth_type == 'digest':
+                        auth = requests.auth.HTTPDigestAuth(config['username'], config['password'])
+                    else:
+                        self.logger.debug('auth_type {} is not supported, defaulting to basic'.format(auth_type))
```

Comment:
```suggestion
                        self.logger.warning('auth_type {} is not supported, defaulting to basic'.format(auth_type))
```
I think we want this to appear on the logs without needing to enable debug